CHANGELOG.md

```diff
@@ -2,5 +2,10 @@
 
 ## 0.5.0
 - Compatible with .NET Standard 2.0
-- Quartz Facilities Quartz supports now Quartz.NET 3.0.2
-- BREAKING CHANGE: namespace is changed into Castle.Facilities.Quartz. Code remains the same
\ No newline at end of file
+- Quartz Facility supports now Quartz.NET 3.0.2
+- BREAKING CHANGE: namespace is changed into Castle.Facilities.Quartz. Code remains the same
+
+## 1.0.0
+- Quartz Facility supports now Quartz.NET 3.0.4
+- C# API added
+- BREAKING CHANGE: Quartz.config is not supported anymore. Listeners can be attached through the C# API. Quartz configuration properties can be attached in the app.config.
```

Comment:
It might be worth also mentioning the interfaces you've removed since they are a breaking change. I'm not using the facility so don't know how important they are to how people could have used this in the past.

Reply:
extra documentation added in changelog.md